<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Regression Suite" duration-ms="45600" started-at="2017-11-11T04:10:03Z" finished-at="2017-11-11T04:10:48Z">
    <groups>
    </groups>
    <test name="verifyLoginWithInvalidCredentails" duration-ms="45600" started-at="2017-11-11T04:10:03Z" finished-at="2017-11-11T04:10:48Z">
      <class name="com.framework.automation.framework.homePage.VerifyloginWithInvalidCredentails">
        <test-method status="FAIL" signature="setup()[pri:0, instance:com.framework.automation.framework.homePage.VerifyloginWithInvalidCredentails@5c3bd550]" name="setup" is-config="true" duration-ms="0" started-at="2017-11-10T23:10:03Z" finished-at="2017-11-10T23:10:03Z">
          <exception class="org.openqa.selenium.WebDriverException">
            <message>
              <![CDATA[Timed out waiting 45 seconds for Firefox to start.
Build info: version: '3.6.0', revision: '6fbf3ec767', time: '2017-09-27T15:28:36.4Z'
System info: host: 'NYCMDLNMUHMAC.local', ip: '2604:2000:a806:6f00:f4cb:6af:487f:3aab', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.4', java.version: '1.8.0_60'
Driver info: driver.version: FirefoxDriver]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.WebDriverException: Timed out waiting 45 seconds for Firefox to start.
Build info: version: '3.6.0', revision: '6fbf3ec767', time: '2017-09-27T15:28:36.4Z'
System info: host: 'NYCMDLNMUHMAC.local', ip: '2604:2000:a806:6f00:f4cb:6af:487f:3aab', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.4', java.version: '1.8.0_60'
Driver info: driver.version: FirefoxDriver
	at org.openqa.selenium.firefox.XpiDriverService.waitUntilAvailable(XpiDriverService.java:112)
	at org.openqa.selenium.firefox.XpiDriverService.start(XpiDriverService.java:97)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:79)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:586)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:217)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:140)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:120)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:98)
	at com.framework.automation.framework.testBase.TestBase.selectBrowser(TestBase.java:41)
	at com.framework.automation.framework.testBase.TestBase.init(TestBase.java:23)
	at com.framework.automation.framework.homePage.VerifyloginWithInvalidCredentails.setup(VerifyloginWithInvalidCredentails.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:514)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:215)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.TestRunner.beforeRun(TestRunner.java:656)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: org.openqa.selenium.net.UrlChecker$TimeoutException: Timed out waiting for [http://localhost:7871/hub/status] to be available after 45003 ms
	at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:100)
	at org.openqa.selenium.firefox.XpiDriverService.waitUntilAvailable(XpiDriverService.java:110)
	... 32 more
Caused by: java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(FutureTask.java:205)
	at com.google.common.util.concurrent.SimpleTimeLimiter.callWithTimeout(SimpleTimeLimiter.java:147)
	at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:75)
	... 33 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.WebDriverException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="SKIP" signature="verifyLoginWithInvalidCredentails()[pri:0, instance:com.framework.automation.framework.homePage.VerifyloginWithInvalidCredentails@5c3bd550]" name="verifyLoginWithInvalidCredentails" duration-ms="2" started-at="2017-11-10T23:10:48Z" finished-at="2017-11-10T23:10:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginWithInvalidCredentails -->
        <test-method status="SKIP" signature="end()[pri:0, instance:com.framework.automation.framework.homePage.VerifyloginWithInvalidCredentails@5c3bd550]" name="end" is-config="true" duration-ms="0" started-at="2017-11-10T23:10:48Z" finished-at="2017-11-10T23:10:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- end -->
      </class> <!-- com.framework.automation.framework.homePage.VerifyloginWithInvalidCredentails -->
    </test> <!-- verifyLoginWithInvalidCredentails -->
  </suite> <!-- Regression Suite -->
</testng-results>
